pyproject.toml

```
@@ -40,7 +40,7 @@ dependencies = [
     "backoff",
     "monotonic",
     # for logging, tracebacks, printing, progressbars
-    "rich <= 13.0.1",
+    "rich",
```

Since the problem is related to this version, what if we force to skip it?
```suggestion
    "rich != 13.1.0",
```

---

I'm down for that as well.